### DISCUSSION TITLE
Any chance of a MacOs ARM version of msft git

### DISCUSSION THREAD

**Discussion by aaronberger-msft** [microsoft/git#553] [General]
Just curious if a MacOs ARM version of MSFT git is at all planned or even feasible?

 We have a pretty large user base working on Apple Silicon macs and there's always a preference for apps in the native arch rather than running through Rosetta.

**Comment by hi-kumar** [comment]
Filed an issue for this: [556](https://github.com/microsoft/git/issues/556)
ARM/AppleSilicon is the primary architecture for new Mac devices. So, releasing git executables for the MacOS ARM platform is very important.

**Comment by keanenwold** [comment]
Hello @aaronbergerM, apologies for the delay and I am duplicating the response to @HiKumar-MS  from issue #556 

We believe the existing intel builds should run without issue and that we do not anticipate any performance differences. We rely heavily on GitHub Actions for Release Engineering and there are not currently any M1 runner available to us. Once M1 runners are available, we can revisit this topic. 

Thank you.
